packages/formation-react/src/components/modal/Modal.jsx

```diff
@@ -41,7 +41,11 @@ class Modal extends React.Component {
 
   teardownModal() {
     if (this.state.lastFocus) {
-      this.state.lastFocus.focus();
+      // Ensure last focus is set before completing modal teardown
+      setTimeout(() => {
+        this.state.lastFocus.focus();
+      }, 0)
+      this.setState({ lastFocus: null })
```

Comment:
this still leads to a problem with the buttons that launch the DD modal :(

`Uncaught TypeError: Cannot read property 'focus' of null`